app/src/main/java/co/touchlab/kampstarter/android/MainActivity.kt

```diff
@@ -18,30 +18,35 @@ class MainActivity : AppCompatActivity(), KoinComponent {
         val TAG = MainActivity::class.java.simpleName
     }
     private lateinit var adapter: MainAdapter
-    private val log:Kermit by inject { parametersOf("MainActivity") }
+    private val log: Kermit by inject { parametersOf("MainActivity") }
 
     private lateinit var viewModel: BreedViewModel
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-
         viewModel = ViewModelProviders.of(this).get(BreedViewModel::class.java)
 
-        viewModel.breedLiveData.observe(this, Observer { itemData ->
-            log.v { "List submitted to adapter" }
-            adapter.submitList(itemData.allItems)
-        })
-        viewModel.errorLiveData.observe(this, Observer { errorMessage ->
-            log.e { "Error displayed: $errorMessage" }
-            Snackbar.make(breed_list, errorMessage, Snackbar.LENGTH_SHORT).show()
-        })
+        viewModel.breedLiveData.observe(
+            this,
+            Observer { itemData ->
```

Comment:
Not for this PR, but I think there's finally a ktx thing for this somewhere so we can drop the `Observer` and do a trailing lambda. Might be misremembering though.